docs/visualization/wcsaxes/overlays.rst

```diff
@@ -60,7 +60,14 @@ in pixel coordinates:
 World coordinates
 *****************
 
-All such Matplotlib commands allow a ``transform=`` argument to be passed,
+The class :class:`~astropy.visualization.wcsaxes.WCSAxes` provides two handy methods:
+:meth:`~astropy.visualization.wcsaxes.WCSAxes.plot_coord`,
+and :meth:`~astropy.visualization.wcsaxes.WCSAxes.scatter_coord`
+
+These are used to plot and scatter respectively :class:`~astropy.coordinates.SkyCoord` or :class:`~astropy.coordinates.BaseCoordinateFrame` coordinates on the axes.
```

Comment:
[docs/common_links.txt](https://github.com/astropy/astropy/blob/8262a315917ca51a01e4cbe63fcf9000048309dd/docs/common_links.txt) defines ReST substitutions, some of which can be used here:
```suggestion
These are used to plot and scatter respectively |SkyCoord| or |BaseFrame| coordinates on the axes.
```